### PR TITLE
fix: move faas_id and cloud_account_id to semantic conventions

### PR DESCRIPTION
**Pull request by svrnm**
Just a minor thing: in the aws-lambda instrumentation two attributes have not been moved to semantic conventions.

### CODE REVIEW

**Comment by svrnm** [comment]
Not sure why lint fails. The error and the suggested changes are not applied when I run `npm run lint:fix` locally and they are also reverted when I apply them manually locally

**Comment by dyladan** [comment]
most likely a version issue. run `lerna clean -y && rm -rf node_modules && git clean -fdx && npm i && npm run compile && npm run lint:fix` then go make yourself a coffee

File: plugins/node/opentelemetry-instrumentation-aws-lambda/src/aws-lambda.ts [hunk 1]
```diff
@@ -121,8 +123,8 @@ export class AwsLambdaInstrumentation extends InstrumentationBase {
         kind: SpanKind.SERVER,
         attributes: {
           [SemanticAttributes.FAAS_EXECUTION]: context.awsRequestId,
-          'faas.id': context.invokedFunctionArn,
-          [CLOUD_RESOURCE.ACCOUNT_ID]: AwsLambdaInstrumentation._extractAccountId(
+          [ResourceAttributes.FAAS_ID]: context.invokedFunctionArn,
+          [ResourceAttributes.CLOUD_ACCOUNT_ID]: AwsLambdaInstrumentation._extractAccountId(
```
Comment (dyladan) [inline plugins/node/opentelemetry-instrumentation-aws-lambda/src/aws-lambda.ts:127]: FWIW this line is the one lint is complaining about most likely (too long)

**Review by dyladan — APPROVED** [review]

**Comment by svrnm** [comment]
@dyladan: I drank some coffee and run that command, thanks for the advice ☕ . Now I have 20 diffs after running `npm lint:fix` looks like the rules for the linter have changed.

**Comment by svrnm** [comment]
Ok, after cleaning up a few more things and making sure that (apparently) my repo is up to date, there are still 20 things to fix, so the rules of the linter have changed and those changes have not yet been applied:

https://github.com/svrnm/opentelemetry-js-contrib/commit/e7c9941d2aabe9d02c07fb9edfd3d2564e400cf0

Since this is not scope of this PR, what's best practice, just have it here or open another PR?

**Comment by dyladan** [comment]
it's fine to just check them in here

**Review by obecny — APPROVED** [review]
lgtm

**Review by Flarna — APPROVED** [review]

**Pull request merged**